slider/client.py

```diff
@@ -678,7 +678,9 @@ def beatmap(self,
     def _parse_user_events(events, _parse_date=_parse_date):
         out = []
         for event in events:
-            event['beatmap_id'] = int(event['beatmap_id'])
+            # beatmap id can be null if the event is a support gift
```

Comment:
Based on the issue, we are guarding against `None`, so we should explicitly say:
```python
if event['beatmap_id'] is not None
```